db/import_listener.go

```diff
@@ -130,35 +130,43 @@ func (il *importListener) StartImportFeed(dbContext *DatabaseContext) (err error
 // internal documents based on key, then checks sync metadata to determine whether document needs to be imported
 func (il *importListener) ProcessFeedEvent(event sgbucket.FeedEvent) (shouldPersistCheckpoint bool) {
 
+	ctx := il.loggingCtx
 	// Ignore non-mutation/deletion events
 	if event.Opcode != sgbucket.FeedOpMutation && event.Opcode != sgbucket.FeedOpDeletion {
 		return true
 	}
-	key := string(event.Key)
+	docID := string(event.Key)
+
+	collection, ok := il.collections[event.CollectionID]
+	if !ok {
+		base.WarnfCtx(ctx, "Received import event for unrecognised collection 0x%x", event.CollectionID)
+		return
```

Comment:
```suggestion
		return true
```
Looks like this should return true to preserve the previous handling.